### PR TITLE
Implement Fizzbuzz using Python

### PR DESCRIPTION
**Pull request by ItIsUday**
# Fizzbuzz

The following snippet is a way to implement the classic Fizzbuzz problem using Python

Related: #15 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

 - [x] My code follows the style guidelines of this project
 - [x] I have performed a self-review of my own code
 - [x] I have commented on my code, particularly in hard-to-understand areas
 - [x] I have made corresponding changes to the documentation
 - [x] My changes generates no new warnings
 - [x] I have added tests that prove my fix is effective or that my feature works
 - [x] New and existing unit tests pass locally with my changes
 - [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: snippets/python/Fizzbuzz.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+2
+Fizz
+.
+.
+98
+Fizz
+Buzz
+```
+
+---
```
Comment (kaustubhgupta) [inline snippets/python/Fizzbuzz.md]: You can remove this link, everything else LGTM 😄

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: snippets/python/Fizzbuzz.md (hunk 1 above)
Comment (ItIsUday) [reply snippets/python/Fizzbuzz.md]: Hey, I have removed the link

**Review by kaustubhgupta — APPROVED** [review]

File: snippets/python/Fizzbuzz.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+1
+2
+Fizz
+.
+.
+98
+Fizz
+Buzz
+```
+
```
Comment (sakshamarora1) [inline snippets/python/Fizzbuzz.md:27]:
```suggestion
```

**Review by sakshamarora1 — COMMENTED** [review]
Looks good! Just a small change.

**Pull request merged**